requirements/edx/base.txt

```diff
@@ -61,7 +60,7 @@ polib==1.0.3
 pycrypto>=2.6
 pygments==2.0.1
 pygraphviz==1.1
-PyJWT==1.4.0
+PyJWT==1.0.1
```

Comment:
@clintonb , we  reverted a few PRs (https://github.com/edx/edx-platform/pull/11324, https://github.com/edx/edx-platform/pull/11311, https://github.com/edx/edx-platform/pull/11241, https://github.com/edx/edx-platform/pull/11302), so I'd expect a few downgrades.